keyboards/hineybush/ibis/keymaps/default/keymap.c

```diff
@@ -0,0 +1,25 @@
+#include QMK_KEYBOARD_H
```

Comment:
```suggestion
// Copyright 2022 Josh Hinnebusch (@hineybush)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```

Reply:
y'all should add this back in to`qmk new_keyboard`

Reply:
We accept PRs to the tooling :stuck_out_tongue_winking_eye: